worldengine/draw.py

```diff
@@ -9,6 +10,36 @@
 # Helper values
 # -------------
 
+### For draw_satellite ###
+NOISE_RANGE = 10 # a random value between -NOISE_RANGE and NOISE_RANGE will be added to the rgb of each pixel
+
+# These are arbitrarily-chosen elevation cutoffs for 4 different height levels. 
+# Some color modifiers will be applied at each level
+HIGH_MOUNTAIN_ELEV = 215
```

Comment:
In other places of the code relative values are used, e.g. "highest 10% of the map". Making use of find_threshold() in basic_operations.py would then lead to the corresponding absolute value.
The used values are stored somewhere, I believe. Something like world.elevation['thresholds'].

I can't verify that myself at the moment, maybe you can take a look.